builtin/providers/google/resource_compute_disk.go

```diff
@@ -185,6 +186,28 @@ func resourceComputeDiskCreate(d *schema.ResourceData, meta interface{}) error {
 	return resourceComputeDiskRead(d, meta)
 }
 
+func resourceComputeDiskUpdate(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	project, err := getProject(d, config)
+	if err != nil {
+		return err
+	}
+
+	if d.HasChange("size") {
```

Comment:
Resizing is an on-line operation. The API docs don't specify the possible states of a disk, and I've never been able to observe another state than 'Ready'.